admin/notifiers/class-stale-content.php

```diff
@@ -0,0 +1,135 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin\Notifiers
+ */
+
+/**
+ * Represents the logic for showing recalibration beta notice.
+ */
+class WPSEO_Stale_Content_Notification implements WPSEO_WordPress_Integration {
+
+	/**
+	 * The name of the notifier.
+	 *
+	 * @var string
+	 */
+	protected $notification_identifier = 'stale-content-notification';
+
+	/**
+	 * Registers all hooks to WordPress
+	 *
+	 * @codeCoverageIgnore
+	 *
+	 * @return void
+	 */
+	public function register_hooks() {
+		add_action( 'admin_init', array( $this, 'handle_notice' ), 15 );
+	}
+
+	/**
+	 * Shows the notification when applicable.
+	 *
+	 * @return void.
+	 */
+	public function handle_notice() {
+		if ( $this->is_applicable( WPSEO_Recalibration_Beta::is_enabled(), WPSEO_Options::get( 'enable_cornerstone_content' ) ) ) {
+			$this->get_notification_center()->add_notification(
+				$this->get_notification()
+			);
+
+			return;
+		}
+
+		$this->get_notification_center()->remove_notification_by_id( 'wpseo-' . $this->notification_identifier );
+	}
+
+	/**
+	 * Returns the notification.
+	 *
+	 * @return Yoast_Notification The notification for the notification center.
+	 *
+	 * @codeCoverageIgnore
+	 */
+	protected function get_notification() {
+		$message  = $this->get_notification_message( WPSEO_Post_Type::get_accessible_post_types() );
+		$message .= ' ';
+		$message .= sprintf(
+			esc_html__(
```

Comment:
Suggest adding a doc line for translators like this:
`/* translators: 1: link opening tag to an article about cornerstone content, 2: link closing tag  */`